### PR TITLE
Update dependencies and increment version to 0.2.6

### PR DESCRIPTION
**Pull request by KennyDizi**
### **PR Type**
Enhancement, Dependencies

___

### **Description**
- Updated `package-lock.json` and `package.json` to reflect new versions of dependencies.
- Incremented project version from `0.2.5` to `0.2.6`.

___

### **Changes walkthrough** 
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Dependencies</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>package-lock.json</strong><dd><code>Updated dependencies and version in `package-lock.json`.</code>&nbsp; </dd></summary>
<hr>

package-lock.json

<li>Updated various dependencies to newer versions.<br> <li> Removed some unused dependencies.<br> <li> Updated <code>version</code> from <code>0.2.5</code> to <code>0.2.6</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/SofarDatas/aws-github-oidc-provider/pull/22/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519">+614/-600</a></td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>package.json</strong><dd><code>Updated dependencies and version in `package.json`.</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

package.json

<li>Updated various dependencies to newer versions.<br> <li> Updated <code>version</code> from <code>0.2.5</code> to <code>0.2.6</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/SofarDatas/aws-github-oidc-provider/pull/22/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519">+15/-15</a>&nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

### CODE REVIEW

**Comment by KennyDizi** [comment]
🏡 🌗 The **[PR description](https://github.com/SofarDatas/aws-github-oidc-provider/pull/22)** has been modified to accurately represent the latest commit (https://github.com/SofarDatas/aws-github-oidc-provider/commit/f3ca0f329d1e293bd4a5ba6a9915707773006308).

**Comment by kenny-cxingularity** [comment]
## PR Analysis 🌵

<table>
<tr><td><br>⏰ &nbsp;<strong>Estimated effort to review</strong>: 1, because the changes are straightforward dependency updates and version increments, which are generally easy to review.
</td></tr>
<tr><td><br>🥇 &nbsp;<strong>Score</strong>: 95, because the PR is well-structured and focuses on updating dependencies and incrementing the version, which are essential maintenance tasks. There are no apparent issues with the changes.
</td></tr>
<tr><td><br>🧪 &nbsp;<strong>No relevant tests</strong></td></tr>
<tr><td><br>🔒 &nbsp;<strong>No security concerns identified</strong></td></tr>
<tr><td><br>🥳 &nbsp;<strong>No key issues to review</strong>: This PR updates various dependencies in 'package-lock.json' and 'package.json' to newer versions and increments the project version from '0.2.5' to '0.2.6'. The updates are essential for maintaining the project and ensuring compatibility with the latest versions of the dependencies.
</td></tr>
</table>

**Comment by KennyDizi** [comment]
## PR Analysis 🌖

<table>
<tr><td><br>⏰ &nbsp;<strong>Estimated effort to review</strong>: 1, because the changes are primarily dependency updates and version increments, which are straightforward to review.
</td></tr>
<tr><td><br>🥇 &nbsp;<strong>Score</strong>: 95, because the PR correctly updates dependencies and increments the version without introducing any apparent issues.
</td></tr>
<tr><td><br>🧪 &nbsp;<strong>No relevant tests</strong></td></tr>
<tr><td><br>🔒 &nbsp;<strong>No security concerns identified</strong></td></tr>
<tr><td><br>🌴 &nbsp;<strong>No key issues to review</strong>: This PR updates various dependencies in 'package-lock.json' and 'package.json' to newer versions and increments the project version from '0.2.5' to '0.2.6'. The changes are well-defined and do not introduce any new functionality or bugs.
</td></tr>
</table>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2024-07-06
+
+### Updated
+- Updated various dependencies including `@aws-sdk/client-iam`, `@aws-sdk/client-sso-oidc`, `@aws-sdk/middleware-logger`, `@smithy/core`, `@smithy/middleware-retry`, `@smithy/middleware-serde`, `aws-cdk-lib`, `cdk-nag`, and others to their latest versions.
+- Increased the project version to `0.2.6`.
```
Comment (KennyDizi) [inline CHANGELOG.md:5]: CHANGELOG.md update

**Pull request merged**